packages/shared/src/common/types/product/product-dto.ts

```diff
@@ -13,6 +13,16 @@ interface Category {
   title: string;
 }
 
+export interface Bid {
+  id: string;
+  bidderId: string;
+  price: number;
+  createdAt: string;
+  updatedAt: string;
+}
+
+type ProductCondition = 'NEW' | 'USED';
+
 type ProductDto = {
```

Comment:
Since we updated ProductDto, it make sense to drop itemDto and create type for productCard

Reply:
Why not to use ProductDto for product card?

Reply:
But we still get ProductDto from Backend in any case when when send request to getById

Reply:
yes, but do we use getById for list?

Reply:
maybe you are right